src/amber/cli/templates/app/config/deploy/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM drujensen/crystal:0.22.0-1
+FROM drujensen/crystal:0.23.1
```

Comment:
can we use the official image?

Reply:
We probably should. It would take some refactoring at a few other places though. When this was built there were a few commands backed into the root dockerfile for dev that didn't make a lot of sense for deployment. Root dockerfile is completely different now though.